backend/app/modules/scheduler/SchedulerActor.scala

```diff
@@ -48,18 +50,29 @@ class SchedulerActor(processes: Queries, configuration: play.api.Configuration,
     * * Download image
     */
   def startProcessing(id: String): Unit = {
-      monitoring.ask(GetProcess(id))
+      val details = monitoring
+        .ask(GetProcess(id))
         .mapTo[Query]
         .map(query => query.details)
         .map {
-          case Some(details) => details
+          case Some(detail) => detail
           case _ =>
             monitoring ! UpdateProcess(id, "failed: no details")
             throw new Exception("no details")
         }
-        .flatMap(details => fetcherActor.ask(FetchRGB("2015-01-01", details.place, Some(10), id)))
+
+      val url = details
+        .flatMap(details => fetcherActor.ask(FetchRGB("2015-01-01", details.place, Some(100), id)))
         .mapTo[FetchResponse]
-        .foreach(response => println("Url from falsk server : " + response.url))
+
+      url.foreach(u => println(u.url))
```

Comment:
A bit odd to see `url.foreach`. What is this? A list of URLs? Add a `s` after `url` then.